examples/sandbox/README.md

```diff
@@ -0,0 +1,29 @@
+# Sandbox
+
+This example is a sandbox for local development. It sets up a basic page
+using Search UI.
+
+It is wired up to use the local source of the search-ui libraries as dependencies via lerna.
+This means as you update library code you can see it updated live in this project.
+
+It is set up to use a pre-configured Elastic App Search engine.
+
+## Using
+
+To run:
+
+```
+# Bootstrapping with lerna will install all dependencies for this entire
+# repository.
+(cd ../.. && npm run bootstrap)
```

Comment:
There could be more description in this section as to where I should start and where I should be when I run these commands.

If I'm in the `search-ui` folder, running the `cd ../..` puts me out in one of my computers containing folders which is not a git repo. Not sure how much of a problem this is for more experienced developers though I could see it being a hangup for less experienced ones.